Lib/fontTools/ttLib/__init__.py

```diff
@@ -203,8 +203,11 @@ def save(self, file, reorderTables=True):
 			# assume "file" is a writable file object
 			closeStream = False
 
-		if self.recalcTimestamp and 'head' in self:
-			self['head']  # make sure 'head' is loaded so the recalculation is actually done
+		if 'head' not in self:
+			raise TTLibError("missing required table: head")
```

Comment:
I wasn't sure either, TBH. This was easier than going through all the places where head is implicitly required and fix them so they continue without it. Maybe that's the right approach..